### PR TITLE
Add test for bug with attribute in WiderFace

### PR DESCRIPTION
**Pull request by yasakova-anastasia**
### Motivation and context

- Update Datumaro version to 0.1.9
- Add test for bug with attribute in WiderFace #2944 

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Comment by nmanovic** [comment]
@yasakova-anastasia , Looks like in different tests we have files with the same name. Could you please fix?

**Comment by yasakova-anastasia** [comment]
@nmanovic , I thought these tests would be in dataset manager server tests. So these tests belong to the same files, they just need to be merged. How should it look in your opinion?

**Comment by nmanovic** [comment]
> @nmanovic , I thought these tests would be in dataset manager server tests. So these tests belong to the same files, they just need to be merged. How should it look in your opinion?

I believe in many cases each test should have own tasks.json and annotations.json. If for a test it isn't important, we can use tasks.json and annotations.json from a common folder. In general I expect that if we add a test, it should not affect another test. Otherwise you have to check each time that previous tests cover fixed problems.

Again, it is a topic for redesign. I don't want to stop current PRs due to the issue.

File: cvat/apps/dataset_manager/tests/test_rest_api_formats.py [hunk 1]
```diff
@@ -189,3 +217,49 @@ def test_api_v1_check_duplicated_polygon_points(self):
...
+    def test_api_v1_check_widerface_with_all_attributes(self):
+        test_name = self._testMethodName
+        dump_format_name = "WiderFace 1.0"
+        upload_format_name = "WiderFace 1.0"
+
+        for include_images in (False, True):
+            with self.subTest():
+                # create task with annotations
+                images = self._generate_task_images(3)
+                task = self._create_task(tasks["widerface with all attributes"], images)
```
Comment (nmanovic) [inline cvat/apps/dataset_manager/tests/test_rest_api_formats.py:230]: In the future here it is better to use "backup" functionality to create a task. Also I don't like that all our tests cases in one file. Probably they should be spitted. What do you think?

**Review by nmanovic — APPROVED** [review]

File: cvat/apps/dataset_manager/tests/test_rest_api_formats.py (hunk 1 above)
Comment (yasakova-anastasia) [reply cvat/apps/dataset_manager/tests/test_rest_api_formats.py:230]: @nmanovic , yes, we probably need to have one file with the main functions for the tests and separate files for the tests.

**Pull request merged**